action.yml

```diff
@@ -56,10 +70,4 @@ runs:
         BASE_OSG_SERIES=${{ inputs.osg_series }}
         TIMESTAMP_TAG=${{ inputs.timestamp_tag }}
       tags: "latest"
-      cache-to: type=local,dest=/tmp/.buildx-cache,mode=max
-
-  - name: Cache Docker images
```

Comment:
Note that you are removing this `Cache Docker images` step that uses `actions/cache@v2`, and you no longer wrap it with `if: ${{ inputs.cache }}` (as you did in one of the earlier commits), so it's not clear to me how disabling the `cache` input option will prevent whatever caching happens in the build-push-action

Reply:
Yeah, this is unfortunately wrong! We still need to keep the "Cache Docker images" step. I had thought that Ryan found that it still worked when referencing a non-existent (or empty) file but if that doesn't work, we'll have to go back to the drawing board